pkg/cmd/drt-run/event.go

```diff
@@ -64,25 +64,25 @@ type Event struct {
 // String() implements the Stringer interface.
 func (e *Event) String() string {
 	var builder strings.Builder
-	fmt.Fprintf(&builder, "[%s] ", e.Timestamp.Format(time.RFC3339))
+	_, _ = fmt.Fprintf(&builder, "[%s] ", e.Timestamp.Format(time.RFC3339))
```

Comment:
I see `fmt.Fprintf` being used in multiple places in the codebase. I think we can remove these changes.